lib/vdsm/virt/livemerge.py

```diff
@@ -287,6 +287,13 @@ def merge(self, driveSpec, base, top, bandwidth, job_id):
             except JobExistsError as e:
                 raise exception.MergeFailed(str(e), job=job.id)
 
+            # Prune stale bitmaps from base volume before measuring.
```

Comment:
This should be called once when starting a merge - good.

But cannot be used when:
- top is the leaf volume - active layer merge, since top can have bitmaps which did not appear in the image yet. New bitamps created when the vm is running will be written to the image only when the vm stop or when we create a snapshot.
- base is not using qcow2 format, prune will fail

So I think we need something like:

    if self._base_needs_prune_bitmaps(drive, job, base_info):
        self._dom._vm.prune_bitmaps(...)